src/plugins/graph/adornments/movable-line/movable-line.tsx

```diff
@@ -129,30 +146,42 @@ export const MovableLine = observer(function MovableLine(props: IProps) {
             x = point.x;
             y = point.y;
           }
-          elt
-            .attr('cx', x)
-            .attr('cy', y);
+          if (x && y) {
```

Comment:
```suggestion
          if (x != null && y != null) {
```
to account for possible zero values